tests/virt_autotest/install_package.pm

```diff
@@ -68,10 +69,10 @@ sub install_package {
         }
     }
 
-    ###SLE-12-SP4 arm64 installation has no KVM role selection
-    if (($repo_0_to_install =~ /SLE-12-SP4/m) && check_var('ARCH', 'aarch64')) {
-        zypper_call("--gpg-auto-import-keys ref",         180);
-        zypper_call("in -t pattern kvm_server kvm_tools", 300);
+    ###Install KVM role patterns form aarch64
+    if (is_remote_backend && check_var('ARCH', 'aarch64')) {
+        assert_script_run("zypper --non-interactive --gpg-auto-import-keys ref",         180);
+        assert_script_run("zypper --non-interactive in -t pattern kvm_server kvm_tools", 300);
```

Comment:
not sure if you had this change included in before but please don't include it in the very same commit, it is not directly related.

Reply:
This part has already been included since a long time ago. The newly and the latest modified part is the comment, which is more accurate now.

Reply:
Please help merge if no other concerns.

Reply:
@okurz Please help merge if no other concerns.

Reply:
Still, please don't include it in the very same commit, it is not directly related. Please use `zypper_call` so better to revert.

Reply:
Done. I updated them all. There is no duplicate content in this commit compared to old implementation. And modified parts in this pull request are integral parts.